packages/quantic/force-app/main/default/lwc/descriptionStrengthIndicator/descriptionStrengthIndicator.js

```diff
@@ -0,0 +1,62 @@
+import { LightningElement, api } from 'lwc';
+
+/**
+ * The `descriptionStrengthIndicator` component is a dynamic indicator that shows the user if their case description has enough details so far. It also give hints to the user as to what information they show include in the description. 
+ * @example
+ * <c-description-strength-indicator></c-description-strength-indicator>
+ */
+export default class DescriptionStrengthIndicator extends LightningElement {
+  /**
+   * The message to be shown to the user.
+   * @api
+   * @type {string}
+   * @defaultValue `'Don’t know what to write?'`
+   */
+  @api helpLabel = 'Don’t know what to write?';
+
+  /** @type {number} */
+  _progression = 0;
+
+  get variant() {
+    return this._progression < 100 ? 'warning' : 'base-autocomplete';
+  }
+
+  get message() {
+    return this._progression === 0 ? 'Provide details' : this._progression < 100 ? 'Provide more details' : 'Thank you!';
```

Comment:
Is the `DescriptionStrengthIndicator` meant to be an example or a reusable component? If it is intended to be a reusable component, I would strongly suggest to support custom thresholds.

Reply:
By custom thresholds you mean that we will have property for this component containing the information of which custom message to show at different  given values of progress right ?

Reply:
Yes, that's the idea. I easily imagine a customer could want an encouraging message around 75-80%. But that depends a lot on how the progress is computed.

I see a few ways we could implement this. On the easy side, we could decide to have 3 thresholds and expose properties to set the message and value of each. On the fancy side, we could have a "threshold" slot and support threshold elements in there. This can become complex quite quickly because of the ordering, overlapping thresholds, gaps between thresholds and such.